controller_interface/include/controller_interface/multi_interface_controller.h

```diff
@@ -172,25 +172,10 @@ std::string enumerateElements(const T& val,
  * };
  * \endcode
  *
- * \tparam T1 Hardware interface type.
+ * \tparam T... Hardware interface types.
  * This parameter is \e required.
```

Comment:
I can use a `static_assert` with `sizeof...` to enforce there always being >=1 interface specified, but I don't actually see a hard requirement anywhere for that being the case. Is it truly illegal to have a "controller" which has no interfaces, but still runs some code on the control thread in its update method?